src/search.js

```diff
@@ -73,6 +74,7 @@ export class SearchElement extends LitElement {
     this.triggerKeycode = 191;
     this.triggerSelector = null;
     this.triggerEvent = "focusin";
+    this.recentSearchesKey = "readthedocs_search_recent_searches"
```

Comment:
```suggestion
    this.recentSearchesLocalStorageKey = "readthedocs-search-recent-searches"
```

Just make the variable's name more explicit and use `-` instead.

Reply:
The reason I used '_` is because local storage allows you to do:

`localStorage.readthedocs_search_recent_searches = 'some-value'`

It's not a feature we need, but that was the reasoning, so normally you'd either do `_` or just use camelCase. But I'm fine leaving it in kebab-case

Reply:
ah, OK. So, using camelCase makes more sense I' say to keep the pattern we are using on the other variables and stuffs